dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/passwordreset/PasswordResetServiceBean.java

```diff
@@ -90,12 +96,36 @@ public PasswordResetInitResponse requestPasswordReset( BuiltinUser aUser, boolea
         
     }
 
+    private void updateTokenExpirationTime(BuiltinUser aUser) {
```

Comment:
We should generally avoid "void" as much as possible, it is harder for testing. If method is updating the user you should return the user. For example `addValues(int firstValue, int secondValue)` should return the int result not void.